libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/view/MapboxRecenterButton.kt

```diff
@@ -145,6 +146,14 @@ class MapboxRecenterButton : ConstraintLayout {
         )?.let { binding.recenterText.setTextColor(it) }
     }
 
+    private fun getExtendToWidth(text: String): Int {
```

Comment:
is it possible to move this function to the shared code?

maybe as extension for textView